dotcom-rendering/src/web/components/Callout/Callout.tsx

```diff
@@ -0,0 +1,112 @@
+import { css } from '@emotion/react';
```

Comment:
This component is the title, description etc as well as the two tabs.  The `CalloutBlockComponent` file imports this component and decides whether it should be collapsible/non collapsible etc